server/superdesk/data_consistency/compare_repositories.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 - 2015 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+
+import superdesk
+import requests
+from superdesk.utc import utcnow
+
+
+class CompareRepositories(superdesk.Command):
+    default_page_size = 500
+
+    def get_mongo_items(self, consistency_record, resource_name):
+        # get the records from mongo in chunks
+        service = superdesk.get_resource_service(resource_name)
+        cursor = service.get_from_mongo(None, {})
+        count = cursor.count()
+        no_of_buckets = len(range(0, count, self.default_page_size))
+        mongo_items = []
+        updated_mongo_items = []
+        for x in range(0, no_of_buckets):
+            skip = x * self.default_page_size
+            print('Page : {}, skip: {}'.format(x + 1, skip))
+            # don't get any new records since the elastic items are retrieved
+            cursor = service.get_from_mongo(None, {'_created': {'$lte': consistency_record['started_at']}})
```

Comment:
Adding a mongo projection would be nice so we retrieve only the `_id` and `_etag`.